src/localization/locales/en.json

```diff
@@ -368,13 +370,20 @@
         },
         "tooltip": "You can add tags and desired properties to your device twin here. To remove a tag or desired property, set the value of the item to be removed to 'null'."
     },
+    "digitalTwin": {
+        "headerText": "Plug and Play (preview)",
+        "dcm": "Model id: {{modelId}}",
+        "componentName": "Component: {{componentName}}",
+        "interfaceName": "Interface: {{interfaceName}}"
+    },
     "deviceProperties": {
         "command" : {
             "refresh": "Refresh",
-            "openReportedValuePanel": "Click to view value"
+            "openReportedValuePanel": "Click to view value",
+            "close": "Close"
         },
         "headerText": "Non-writable properties",
-        "noProperty": "No properties",
+        "noProperties": "Interface {{interfaceName}} contains no non-writable property definition",
```

Comment:
should "non-writable" be "read-only"?
should "definition" be plural "definitions"?

Reply:
The non-writable is the term pnp uses in external docs, so I am keeping it this way.